src/nxt_router.c

```diff
@@ -1613,6 +1617,13 @@ static nxt_conf_map_t  nxt_router_websocket_conf[] = {
 };
 
 
+#if (NXT_HAVE_OTEL)
+static void nxt_otel_log_callback(u_char *arg)
+{
+    printf("otel: %s", (char *) arg);
+}
+#endif
```

Comment:
Some left over debugging?

Reply:
No in fact, this log callback is used to handle any error output at time of initialization. Currently the Rust code has no way of hooking into the Unit logging code.

If you prefer, I would be in favor of removing it and simply using stdout/stderr from the Rust side of things, but I do not currently see a path to get the Unit logging features working in the Rust codebase. I dont think wasm-wasi-component use them either.

Reply:
OK, then I guess it should be using `nxt_log()`

Reply:
Let's just export this function from `nxt_otel.c`, the less we need to touch the unit core the better...

We won't need the #if/#endif then, but of course it should be formatted as

```c
void 
nxt_otel_log_callback(u_char *arg)
{
    nxt_log(...);
}
```
Oh and if you're wondering how you get `task`

```c
nxt_thread_t  thr;

thr = nxt_thread();

nxt_log(thr->task, ...);
```